examples/svdiff/train_svdiff.py

```diff
@@ -0,0 +1,1112 @@
+#!/usr/bin/env python
+# coding=utf-8
+# Copyright 2023 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
```

Comment:
I know all the licensing entries in our training scripts are incomplete, but could you add a complete entry like so?
 
https://github.com/huggingface/diffusers/blob/262d539a8a8f505dc72958f7ea50915a4b56dfac/src/diffusers/models/attention_processor.py#LL1C1-L13C33

Reply:
fixed https://github.com/huggingface/diffusers/pull/3107/commits/7ef4bc066dd04273333507fc5cdaf7d33c59385f